.github/workflows/pre-review.yml

```diff
@@ -108,12 +124,11 @@ jobs:
       - name: run unit tests
         id: unitTest
         run: ./gradlew $GRADLEW_UNIT_TEST_ARGS
-      - name: Publish Test Report
-        uses: mikepenz/action-junit-report@5f47764eec0e1c1f19f40c8e60a5ba47e47015c5
-        if: success() || failure() # always run even if the build step fails
```

Comment:
so Publish Test Report doesn't happen any more?

Reply:
correct.  annotating the PR requires write access to the repository that the PR lives in, which is the targeted fork. If we want to allow this, we would need to switch the event type back to `pull_request_target` which would then run in the context of the targeted fork, using the version of the workflow defined there.